stable/gcloud-sqlproxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: gcloud-sqlproxy
-version: 0.2.0
+version: 0.2.1
```

Comment:
This is a version bump to the gcloud-sqlproxy chart. Yet, there are no changes to this chart in the PR.